henryade/Banka-React
# #168078528 Implement Reset Password Feature

#### What does this PR do?
Implement the rest password feature
#### Description of Task to be completed?
- add components and styles
- add actions, action types and reducers
- add custom functions
#### How should this be manually tested?
Open [Link](https://banka-app-in-react.herokuapp.com/reset) to see page.
Input your email and you will be sent an email with a reset link.
The link directs you to the change password page where you can change your password
#### Any background context you want to provide?
 - N/A
#### What are the relevant pivotal tracker stories?`
[#168078528](https://www.pivotaltracker.com/story/show/168078528)
#### Screenshots (if appropriate)
##### Reset Password:
![Screenshot from 2019-11-25 17-40-05](https://user-images.githubusercontent.com/47576037/69563133-cb700b80-0fb0-11ea-9922-9f6f96cb9e43.png)

![Screenshot from 2019-11-25 17-40-25](https://user-images.githubusercontent.com/47576037/69563522-8dbfb280-0fb1-11ea-8ce5-344f6d999f9b.png)

##### Change Password:
![Screenshot from 2019-11-25 18-05-20](https://user-images.githubusercontent.com/47576037/69563550-9b753800-0fb1-11ea-9f58-e8c5fa4d4136.png)

![Screenshot from 2019-11-25 18-23-34](https://user-images.githubusercontent.com/47576037/69563555-9f08bf00-0fb1-11ea-8337-bf54f85df6bd.png)
